terraform/modules/swipe-sfn-batch-job/variables.tf

```diff
@@ -70,3 +70,13 @@ variable "docker_network" {
   type        = string
   default     = ""
 }
+
+variable "sfn_notification_queue_arns" {
+  description = "ARNs of notification SQS queues"
+  type        = list(string)
+}
```

Comment:
It seems like these are not used. If you go the SNS route it will be by ARN not URL and you will only need one.